elytron/WFCORE-3970-reload-ldap-key-store.adoc

```diff
@@ -0,0 +1,58 @@
+= WFCORE-3970 Add support for Elytron's ldap-key-store to be dynamically reloaded
+:author:            Justin Michael Cook
+:email:             jucook@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+It is not currently possible to reload the certificates dynamically for Elytron's ldap-key-store, requiring WildFly to be restarted in order to see any changes done in the LDAP directory. This feature is currently available for simple file based keystores with the load operation. This task is to add similar support for the ldap-key-store.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-3970[WFCORE-3970]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1052[EAP7-1052]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:msvehla@redhat.com[Martin Švehla]
+
+=== Affected Projects or Components
+
+* WildFly, Security
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* When changes are made to certificates in the LDAP directory, it should be possible to reload these certificates dynamically without having to restart the server. This should be done by adding a load operation to the Elytron subsystem's ldap-key-store resource. This new management operation will not take any parameters.
```

Comment:
Just a minor note, it would be good to mention that the new management operation will be a runtime-only operation like the existing key-store load operation.

Reply:
Updated.